London class 8/margarita glushkova/grid_project_juice_bar

RitaGlushkova opened the pull request:
Only one page is in the process - the landing page. The Mobile version looks similar, the desktop version is not complete. Still need to do the iPad version.
Still have to do Menu Page - all media Q.

RitaGlushkova committed:
html_first_draft

Index page html, first draft

RitaGlushkova committed:
mobile_landing_page_draft

RitaGlushkova committed:
landing_page_mobile

All elements are now in place and styled.

But adjustments might be needed later when designing desktop version.

RitaGlushkova committed:
Landing made_desktop draft

The draft is not complete.
Still need to style a few sections and hovers.

LucyMac commented on css/common.css:
```diff
@@ -0,0 +1,311 @@
+  --buttons-font: "Raleway", sans-serif;
+  --body-font: "Roboto", sans-serif;
+}
+.pad_main {
+  padding: 0 30px;
+}
+.mar_bot {
+  margin-bottom: 30px;
+}
+.pad_bot {
+  padding-bottom: 24px;
+}
+.pad_top {
+  padding-top: 33px;
+}
```
This is fine. You could also save these values inside variables, and apply them to other classes when needed, e.g.
```
.some-other-element {
     padding: var(--space-small);
}
```

LucyMac commented on css/common.css:
```diff
@@ -0,0 +1,311 @@
+  grid-template-areas:
+    "order order order"
+    "bar . hamburger";
+  align-items: center;
```
You might want to add a `z-index` here to make sure the header doesn't slide underneath the 'follow' section at the bottom.

LucyMac commented on index.html:
```diff
@@ -1,21 +1,331 @@
+    <!--Main-->
+    <main>
+      <!--Section Explore-->
+      <section class="explore">
+        <h1 class="text_center width_adjust margin_auto font_1">
+          Fuel Your Body. Balance Your Mind.
+        </h1>
+        <p class="font_2 text_center width_adjust margin_auto">
+          Organic Cold Pressed Juices & Smoothies
+        </p>
+        <!--Link Explore-->
+        <a href="#" class="button_black flex_center margin_auto no_underline"
+          >Explore Flavors</a
+        >
+      </section>
```
This doesn't matter here, but just to clarify what a 'hero' is - I would argue that this section is the hero, _as well as_ the image below it.

LucyMac commented on index.html:
```diff
@@ -1,21 +1,331 @@
+            <div class="feature-img">
+              <img src="img/Nat_ingr.svg" />
+            </div>
+            <h3 class="font_2">100% Natural Ingredients</h3>
```
These `h3` (like all other `h1-6` headings) have default margin set on them by the browser. You'll need to override this by setting your own margin (at the moment there's too much space between this heading and the paragraph below it)

LucyMac commented on css/common.css:
```diff
@@ -0,0 +1,311 @@
+  width: 160px;
+  height: 42px;
+  border: solid 2px var(--black);
+  background-color: none;
```
Really like your buttons 💅 

2 things: 
1) Avoid giving buttons a set width as it stops them from growing when their content changes. They should be able to grow / shrink with their content. The way to make them look consistent is by setting padding, but no width. This will ensure there's always the same spacing around the word(s). The same applies to height. 
2) `background-color: none` shows up as invalid in the inspector. You can set the color to `transparent` instead.

LucyMac commented on css/index_style.css:
```diff
@@ -0,0 +1,206 @@
+.follow img {
+  width: 100%;
+}
+@media screen and (min-width: 481px) {
```
481px is a little too early to be switching out of mobile view. I'd keep mobile view until 600px minimum at this stage. 
Where you decide to 'break' will depend on how many breakpoints you have (i.e. 1 breakpoint = only 2 versions, therefore better to break later. Whereas 4 breakpoints = 5 versions and therefore you'll spread the breakpoints out more).

LucyMac commented on css/index_style.css:
```diff
@@ -0,0 +1,206 @@
+    width: 490px;
+  }
+  .vision img {
+    width: 557px;
```
You're setting quite a few fixed widths which I wouldn't recommend. These mean your layouts will look perfect at a very particular screen size, but won't adapt to for example a slightly larger phone, or a slightly smaller desktop. 
It's best to set widths either in `%` or in `fr` if using grid. This ensures you are setting proportions instead of fixed dimensions. In the case of this image, it's basically taking up half of the screen in desktop view, so that's what you should be aiming for rather than a fixed width.

LucyMac commented on index.html:
```diff
@@ -1,21 +1,331 @@
+      <!--Section Follow Us-->
+      <section class="text_center">
+        <h2 class="font_1">Follow Us</h2>
+        <section class="follow">
```
This section looks awesome, well done 🌟

LucyMac reviewed (commented):
Really great work on the mobile view @RitaGlushkova, it looks awesome. Great HTML structure and spotless styling ✨ I've left a few comments about keeping the styling more fluid by default rather than fixing it to the current screen size.
Looking forward to seeing it fully responsive 😃

RitaGlushkova committed:
Hover discover follow sections for desktop

added style for desktop version for sections: footer, follow and discover.

Hovers added.

All for landed page only.

RitaGlushkova commented:
@LucyMac Hi Lucy thank you very much for taking your time to review my work. Your feedback is very helpful. I am planning to finish this project by the end of January. I will send you an update once it is done :)
